content/dessert/fresh-apple-cake.md

```diff
@@ -0,0 +1,75 @@
+---
+title:  Fresh Apple Cake"
+category: "dessert"
+date: 2024-02-06
+recipeYear: YYYY
```

Comment:
When there's no value for `recipeYear` we need to remove the placeholder so that "YYYY" isn't set as the recipe year and instead the computer understands that there's just no data.

```suggestion
recipeYear:
```